### DISCUSSION TITLE
Not able to arr containers

### DISCUSSION THREAD

**Discussion by curtis-brown100** [qdm12/gluetun#2286] [Q&A]
I am running Gluetun on portainer.  I have put SABnzbd behind the VPN.  After this is completed, I try to set up Sonarr/Radarr.  When setting up the download clients, I was to connect to SABnzbd.  However Sonarr is not able to connect to Sabnzbd.  

I am not running Sonarr thru the vpn, just SABnzbd.  

Any ideas would be appreciated.

**Comment by ppriorfl** [comment]
What do you have entered into Sonarr for download client settings?    Are all your dockers in the same stack?  I use host IP and port for the Sabnzbd which is the gluetun forwarded one and it connects to sabnzbd without issue.
